diff --git a/libsqsh/src/easy/traversal.c b/libsqsh/src/easy/traversal.c
index 458ddcf..fa05974 100644
--- a/libsqsh/src/easy/traversal.c
+++ b/libsqsh/src/easy/traversal.c
@@ -34,2 +34,2 @@
-#include "sqsh_tree.h"
 #define _DEFAULT_SOURCE
+#include "sqsh_tree.h"
diff --git a/libsqsh/src/posix/file_ext.c b/libsqsh/src/posix/file_ext.c
index f2ae125..6e394d2 100644
--- a/libsqsh/src/posix/file_ext.c
+++ b/libsqsh/src/posix/file_ext.c
@@ -108 +108 @@ iterator_worker(void *data) {
-	int rv = 0, rv2 = 0;
+	int rv = 0;
@@ -131,2 +131,0 @@ out:
-	assert(rv2 == 0);
-
@@ -180 +178 @@ file_iterator_mt(
-			calloc(sizeof(struct FileIteratorMtBlock), (size_t)block_count);
+			calloc((size_t)block_count, sizeof(struct FileIteratorMtBlock));
@@ -210 +208 @@ sqsh_file_iterator_mt(
-	struct FileIteratorMt *mt = calloc(sizeof(struct FileIteratorMt), 1);
+	struct FileIteratorMt *mt = calloc(1, sizeof(struct FileIteratorMt));
@@ -256 +254 @@ sqsh_file_to_stream_mt(
-	struct FileToStreamMt *mt = calloc(sizeof(struct FileToStreamMt), 1);
+	struct FileToStreamMt *mt = calloc(1, sizeof(struct FileToStreamMt));
